lib/pf/UnifiedApi/Controller/Dhcpoption82s.pm

```diff
@@ -0,0 +1,52 @@
+package pf::UnifiedApi::Controller::Dhcpoption82s;
+
+=head1 NAME
+
+pf::UnifiedApi::Controller::Dhcpoption82s -
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::UnifiedApi::Controller::Dhcpoption82s
+
+=cut
+
+use strict;
+use warnings;
+use Mojo::Base 'pf::UnifiedApi::Controller::Crud';
+use pf::dal::dhcp_option82;
+
+has dal => 'pf::dal::dhcp_option82';
+has url_param_name => 'dhcpoption82_id';
```

Comment:
dhcp_option82_id to stay consistent

Reply:
However if really needs to be it as dhcp_option82_id then change then rename the controller to lib/pf/UnifiedApi/Controller/Dhcp_option82s.pm.

Then update the documentation
Because the path will change from /dhcpoption82s -> /dhcp_option82s and /dhcpoption82/:dhcpoption82_id -> /dhcp_option82/:dhcp_option82_id

Reply:
@julsemaan go, or no-go on `dhcp_option82` ?

Reply:
I'd say go for dhcp_option82_id so that its consistent with the table name which is called dhcp_option82

Reply:
Sorry it is better to rename the controller this way 
Dhcpoption82s -> DhcpOption82s (just uppercase the O in option)
instead of my first suggestion.